bin/utils.py

```diff
@@ -375,15 +375,20 @@ def upload_rsync(self, fr, to, flags = []):
   def upload(self, fr, to, type):
     self.copy(fr, "%s/%s/%s" % (self.location_root, type, to))    
 
-def maven_build_distribution(version):
+def maven_build_distribution(version, args):
   """Builds the distribution in the current working dir"""
   mvn_commands = [["clean"], ["install"], ["deploy"]]
     
   for c in mvn_commands:
+    mvn_command = c[0]
     if settings['dry_run']:
       c.append("-DskipNexusStagingDeployMojo=true -Dmaven.deploy.skip=true")
     if not settings['verbose']:
       c.insert(0, '-q')
+    if mvn_command == 'install' and args.assetRepo and args.assetBranch: 
+      mvn_param = '-Dbranding.repo={assetRepo} -Dbranding.branch={assetBranch}'
+      mvn_param = mvn_param.format(assetRepo=args.assetRepo, assetBranch=args.assetBranch)
```

Comment:
This can be a oneliner and you can just use `{}` as only two args, so: `"-Dbranding.repo={} -Dbranding.branch={}".format(args.assetRepo, args.assetBranch)`. Also note that we should use double quotes.

Reply:
Ok +1